test/integration/guest_user_test.rb

```diff
@@ -3,11 +3,13 @@
 class GuestUserTest < ActionDispatch::IntegrationTest
   include Capybara::DSL
 
-  attr_reader :category, :item
+  attr_reader :item1, :item2, :category1, :category2
 
   def setup
-    @item = Item.create(name: 'espresso', cost: 9000)
-    @category = item.categories.create(name: 'hot beverages')
+    @item1 = Item.create(name: 'espresso', cost: 9000)
+    @category1 = item1.categories.create(name: 'hot beverages')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.